docs/content/en/blog/articles/dev-config/index.md

```diff
@@ -0,0 +1,321 @@
+
+We can create a secret for Medusa by applying the following yaml:
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+ name: medusa-bucket-key
+type: Opaque
+stringData:
+ # Note that this currently has to be set to medusa_s3_credentials!
+ medusa_s3_credentials: |-
+   [default]
+   aws_access_key_id = <aws key>
+   aws_secret_access_key = <aws secret>
+```
+
+
+You’ll notice in our Helm values that they lack heap settings. We intentionally did this to set them when invoking `helm install` with various heap values for our different tests.
+
+To fully set up our environment, we executed the following steps:
+
+1. Create the kind cluster: 
+    ```
+    kind create cluster --config ./kind.config.yaml
+    ```
+1. [Configure and install Traefik]({{< relref "docs/topics/ingress/traefik/kind-deployment/#3-create-traefik-helm-values-file">}})
+1. Create a namespace: 
+    ```
+    kubectl create namespace k8ssandra
+    ```
+1. (If Medusa is enabled) Create the secret: 
+    ```
+    kubectl apply -f medusa_secret.yaml -n k8ssandra
+    ```
+1. Deploy K8ssandra with the desired heap settings: 
+    ```
+    helm repo add k8ssandra https://helm.k8ssandra.io/stable
+    helm repo update
+    helm install k8ssandra k8ssandra/k8ssandra -n k8ssandra \
+        -f /path/to/three_nodes_cluster_with_stargate.yaml \
+        --set cassandra.heap.size=500M,cassandra.heap.newGenSize=250M,stargate.heapMB=300
+    ```
+
+
+You’ll have to wait for the `cassandradatacenter` resource and then the Stargate pod to be ready before you can start interacting with Cassandra. This usually takes around 7 to 10 minutes.
+
+You can wait for the `cassandradatacenter` to be ready with the following `kubectl` command:
+
+```
+kubectl wait --for=condition=Ready cassandradatacenter/dc1 --timeout=900s -n k8ssandra
+```
+
+Then wait for Stargate to be ready:
+
+```
+kubectl rollout status deployment k8ssandra-dc1-stargate -n k8ssandra
+```
+
+Once Stargate is ready, the above command should output something like this: 
+
+```
+deployment "k8ssandra-dc1-stargate" successfully rolled out.
+```
+You can execute a NoSQLBench stress run by creating a k8s job. You’ll need the superuser credentials so that NoSQLBench can connect to the Cassandra cluster.
+You can get those credentials with the following commands ( requires [`jq`](https://stedolan.github.io/jq/) to be installed):
+
+```
+SECRET=$(kubectl get secret "k8ssandra-superuser" -n k8ssandra -o=jsonpath='{.data}')
+echo "Username: $(jq -r '.username' <<< "$SECRET" | base64 -d)"
+echo "Password: $(jq -r '.password' <<< "$SECRET" | base64 -d)"
+```
+
+Then create the NoSQLBench job which will start automatically:
+
+```
+kubectl create job --image=nosqlbench/nosqlbench nosqlbench -n k8ssandra \
+    -- java -jar nb.jar cql-iot rampup-cycles=1k cyclerate=100 \
```

Comment:
Is it possible to remove the `java -jar nb.jar` since that's the entrypoint?

```suggestion
    -- cql-iot rampup-cycles=1k cyclerate=100 \
```

Reply:
@adejanovski

Reply:
Actually not, it doesn't work if you don't put this all with `java -jar nb.jar...`. I got this tip from the Fallout team as I was struggling to get it to work.